### PR TITLE
User Profile docs

### PR DESCRIPTION
**Pull request by shawngustaw**
Refs #SDE-????

TODO:
- [ ] Set milestone label in pull request (SDE version)
- [ ] Updated changelog.md
- [ ] Completed documentation for the new endpoint.
- [ ] Feature has been merged into SDE develop.

### CODE REVIEW

File: source/includes/user_profiles.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Authorization: Token "YOUR SDE ACCESS TOKEN"
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "id": 1,
+  "first_name": "John",
```
Comment (shawngustaw) [inline source/includes/user_profiles.md]: These are already exposed by User's endpoint so don't need to duplicate it here.

File: source/includes/user_profiles.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  "last_name": "Doe",
+  "plaintext_emails_only": false,
+  "notify_on": {
```
Comment (shawngustaw) [inline source/includes/user_profiles.md]: Just for reference, these are enumerated on the UserProfile model: users/models/profile.py line 175

**Comment by hanifvirani** [comment]
Can you update the PR name.

File: source/includes/users.md [hunk 3]
```diff
@@ -222,6 +222,54 @@ user_id   | The ID of the User to update
...
+Authorization: Token "YOUR SDE ACCESS TOKEN"
+```
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "id": 1,
+  "first_name": "John",
```
Comment (hanifvirani) [inline source/includes/users.md]: Let's remove the `first_name` and `last_name` from here. Probably don't need `id` either.

**Comment by hanifvirani** [comment]
This should also be on the `/users/me/profile`

**Comment by funkaoshi** [comment]
Mention change in `changelog.md`.

**Comment by hanifvirani** [comment]
I have updated this doc endpoint with the `notify_tasks` attribute.

**Comment by shawngustaw** [comment]
I've merged this into #130

**Pull request closed without merging**